Docs/source/usage/parameters.rst

```diff
@@ -801,6 +801,16 @@ Particle initialization
 
       If ``<species_name>.focal_distance`` is specified, ``x_rms``, ``y_rms`` and ``z_rms`` are the size of the beam in the focal plane. Since the beam is not necessarily initialized close to its focal plane, the initial size of the beam will differ from ``x_rms``, ``y_rms``, ``z_rms``.
 
+      Usually, in accelerator physics the operative quantities are the emittance :math:`\epsilon_{x,y}` and beta function :math:`\beta_{x,y}`.
+      They are related to the focal distance :math:`f`, the beam size :math:`\sigma_{x,y}(z)` (at focus :math:`\sigma_{0x,0y}`), the energy of the beam :math:`\gamma`, and the normalized momentum spread :math:`du_{x,y}` according to the following equations:
```

Comment:
The nomenclature between `\sigma_{0x,0y}` and input parameter `x_rms`, etc. is not really explicit. Are they the same?